db/model/Gdoc/archieToGdoc.ts

```diff
@@ -212,19 +213,51 @@ export async function createGdocAndInsertOwidArticleContent(
         version: "v3",
         auth,
     })
-    const createResp = await driveClient.files.create({
-        supportsAllDrives: true,
-        requestBody: {
-            parents: [targetFolder],
-            mimeType: docsMimeType,
-            name: content.title,
-        },
-        media: {
-            mimeType: docsMimeType,
-            body: "",
-        },
-    })
-    const documentId = createResp.data.id!
+    let documentId = existingGdoc
+    if (existingGdoc) {
```

Comment:
This section had me going up and down a lot understand what was going on. It'd be nice if you could lift the branches up into description functions 🙂 

e.g.
```js
  await deleteGdocContents()
} else {
  await createNewGdoc()
}
await setGdocContents()
return documentId
```